src/Models/ActionModel.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+const actionSchema = new mongoose.Schema({
+  actionMessage: {
+    required: true,
+    type: String,
+    minlength: 8,
+    maxlength: 180
+  }
```

Comment:
For timeseries data I would always add a date field that adds the current timestamp automatically.